src/Chainweb/Miner/Coordinator.hs

```diff
@@ -274,8 +275,8 @@ estimatedHashes (PrevTime p) b = floor $ (d % t) * 1000000
     d = case targetToDifficulty $ _blockTarget b of
         HashDifficulty (PowHashNat w) -> int w
 
-getAdjacentParents :: Cut -> BlockHeader -> Maybe BlockHashRecord
-getAdjacentParents c p = BlockHashRecord <$> newAdjHashes
+getAdjacentParents :: Cut -> ParentHeader -> Maybe BlockHashRecord
+getAdjacentParents c (ParentHeader p) = BlockHashRecord <$> newAdjHashes
```

Comment:
If we increase the usage of this pattern, we would likely reduce confusion in deep code locations where both the parent and child headers are passed around beside one another.